### DISCUSSION TITLE
Apakah akan diadakan fitur peleburan asset?

### DISCUSSION THREAD

**Discussion by sukabumikode** [ruswan/app-management#29] [Q&A] [answered]
Karena asset rata-rata bertahan sekitar 5 tahun lalu perlu diperbaiki

**Comment by ruswan** [comment, marked as answer]
Untuk saat ini nampaknya belum, tetapi apabila hendak berkontribusi dan menambahkan fitur tersebut, kami akan sangat berterimakasih

**Reply by sukabumikode** [reply]
Baik, terimakasih
